docs/src/content/docs/reference/frontmatter.md

```diff
@@ -206,6 +206,13 @@ title: Page with a custom last update date
 lastUpdated: 2022-08-09
 ---
 ```
+```md
+---
+# src/content/docs/example.md
+title: Page with disabled lastUpdated global config
```

Comment:
I think this title might make more sense here. Using `lastUpdated: false` in a page would only really make sense if the global config is enabled.

```suggestion
title: Page with a hidden last-updated date
```